docs/source/adventure/basics.rst

```diff
@@ -0,0 +1,691 @@
+              self.wait(0.3)
+
+
+-------------------
+``.animate`` Syntax
+-------------------
+
+One of the most powerful features of Manim is it's ``.animate`` syntax. It allows you to animate the changing of an attribute of a mobject. You can see an example below:
+
+.. manim:: AnimateSyntaxExample
+
+   class AnimateSyntaxExample(Scene):
+      def construct(self):
+          c = Circle()
+          self.add(c)
+          self.play(c.animate.shift(RIGHT))
+          self.play(c.animate.to_corner(DL).set_fill(color=RED, opacity=0.4))
+
+
+
+.. note::
+
+   ``.animate`` works by interpolating between the initial and the final mobject. As such, beware when using ``.animate.rotate`` with angles greater than pi radians
+   as it may not produce the intended animation.
+
+
+-------------------
+Test Your Knowledge
+-------------------
+
+Try to create the following animation!
+
+.. manim:: TestBasicAnimationKnowledge
+    :hide_source:
+
+    class TestBasicAnimationKnowledge(Scene):
+        def construct(self):
+            c = Circle().set_fill(color=RED, opacity=0.5)
+            s = Star().set_stroke(color=YELLOW).set_fill(color=YELLOW, opacity=0.3)
+            t = Triangle().set_fill(color=BLUE, opacity=0.1)
+            VGroup(c, s, t).arrange(RIGHT).move_to(ORIGIN) # users will arrange manually
+            self.play(
+                DrawBorderThenFill(c),
+                GrowFromPoint(s, ORIGIN),
+                SpinInFromNothing(t),
+                run_time=2
+            )
+            self.wait()
+            for mob in (s, t):
+                self.play(Transform(c, mob))
+                self.remove(mob)
+                self.wait(0.2)
+            self.play(c.animate.move_to(ORIGIN))
+
+Hint: you might need to look at different :doc:`/reference_index/animations`!
+
+=================
+Grouping Mobjects
+=================
+
+Oftentimes it is convenient to animate the movement of several mobjects at once. To help accomplish this goal, manim provides two classes: ``Group`` and ``VGroup``.
+99% of the time, ``VGroup``'s are used, but if you're dealing with some form of an ``ImageMobject`` you will have to use ``Group``. Here's an example of how groups can be useful:
+
+.. manim:: GroupingExample
+
+    class GroupingExample(Scene):
+        def construct(self):
+            tri = Triangle()
+            sq = Square()
+            circ = Circle()
+            grp1 = VGroup(tri,sq,circ).arrange(RIGHT)
+            grp2 = VGroup(tri,circ)
+            self.add(tri,sq,circ)
+            self.play(grp1.animate.shift(UP))
+            self.play(grp2.animate.shift(2*DOWN))
+            self.play(tri.animate.next_to(circ,RIGHT))
+            self.play(grp1.animate.shift(UP))
+            self.wait()
+
+.. note::
+   From now onwards, if we refer to a group we are referring to a ``VGroup``, unless specifically stated otherwise.
+
+Groups also have a bunch of methods to make your life easier. Take a look at some in the example below:
```

Comment:
Maybe a bit more text to get the point across, additonally i would also state that animations work on groups as well as all the other mobjects because they are indifferent to each other